fastparquet/writer.py

```diff
@@ -944,18 +1019,60 @@ def write(filename, data, row_group_offsets=50000000,
                 raise ValueError('Requested file scheme is %s, but '
                                  'existing file scheme is not.' % file_scheme)
             fmd = pf.fmd
-            i_offset = find_max_part(fmd.row_groups)
+            if rgp_by_date:
+                if date_col:
+                    # Use date column provided.
+                    date_index_name = date_col
+                else:
+                    # Date index is 1st column
+                    date_index_name = index_cols[0]
+
+                # Formatting `drop_duplicates_on` for use in `pd.concat()`.
+                if drop_duplicates_on:
```

Comment:
Hmmmm, I need it, otherwise I am sure to have redundant data.
I will do it if you confirm you really need it as a separate PR, but basically, the date row grouping is not usable for me without the drop duplicate, so I will resubmit both at the same time.